gen/src/builtin.rs

```diff
@@ -219,15 +218,6 @@ pub fn write(out: &mut OutFile) {
         writeln!(out, "using Fat = ::std::array<::std::uintptr_t, 2>;");
     }
 
-    if builtin.ptr_len {
```

Comment:
all this code migrated to cxx.h/cxx.cc - there's no reason for us to keep generating it.